uiplib/UIP.py

```diff
@@ -46,12 +45,7 @@ def main():
     if settings['offline']:
         print("You have choosen to run UIP in offline mode.")
     if settings['flush']:
-        print("Deleting all downloaded wallpapers...")
-        try:
-            shutil.rmtree(settings['pics-folder'])
-            make_dir(settings['pics-folder'])
-        except FileNotFoundError:
-            pass
+        flush_wallpapers(settings)
```

Comment:
i guess you should just pass pics-folder, no point in passingall of the settings